src/pages/dmt5/_download-app.tsx

```diff
@@ -268,6 +270,14 @@ const DownloadApp = () => {
                             >
                                 <img src={AppGallery} alt="app gallery" width="132" height="40" />
                             </LocalizedLink>
+                            <LocalizedLink
+                                external
+                                to={dmt5_ios_url}
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                <img src={Mac} alt="app gallery" width="132" height="40" />
```

Comment:
Please use approach with `StaticImage` 🙏 
https://deriv-group.slack.com/archives/C02CTVA0LBS/p1677037272787159
Also we can replace similar places in this file and it'll help to increase performance of our app 😃 
```suggestion
                                <StaticImage
                        src="../../images/svg/dmt5/dmt5_app_store.svg"
                        alt="app gallery"
                        loading="eager"
                        formats={['avif', 'webp', 'auto']}
                        quality={30}
                        objectFit="contain"
                        placeholder="none"
                        width="132" 
                        height="40"
                    />
```